backend/editor/models.py

```diff
@@ -16,3 +16,70 @@ class Header(Marginal):
 
 class Footer(Marginal):
     pass
+
+
+# Models for FastAPI
+
+
+class ImportFromGithubParameters(BaseModel):
+    branch: str
+    taxonomy_name: str
+
+
+class ImportFromGithubResponse(BaseModel):
+    status: str
+
+
+class CreateNodeParameters(BaseModel):
+    branch: str
+    taxonomy_name: str
+
+
+class EditEntryParameters(BaseModel):
+    branch: str
+    taxonomy_name: str
+    entry: str
+
+
+class EditEntryResponse(BaseModel):
+    result = []
+
+
+class EditChildrenParameters(BaseModel):
+    branch: str
+    taxonomy_name: str
+    entry: str
+
+
+class EditChildrenResponse(BaseModel):
+    result = []
+
+
+class EditSynonymParameters(BaseModel):
+    branch: str
+    taxonomy_name: str
+    entry: str
+
+
+class EditSynonymResponse(BaseModel):
+    result = []
+
+
+class EditHeaderParameters(BaseModel):
+    incoming_data: Header
+    branch: str
+    taxonomy_name: str
+
+
+class EditHeaderResponse(BaseModel):
+    result: []
+
+
+class EditFooterParameters(BaseModel):
+    incoming_data: Footer
+    branch: str
+    taxonomy_name: str
+
+
+class EditFooterResponse(BaseModel):
+    result: []

```

Comment:
The point of the issue was to be able to have a good documentation.

So we should add descriptions. (see https://fastapi.tiangolo.com/tutorial/query-params-str-validations/#declare-more-metadata)

see also [comment above](https://github.com/openfoodfacts/taxonomy-editor/pull/157#discussion_r1059996463) to avoid repeating comments